vizro-core/examples/visual-vocabulary/custom_charts.py

```diff
@@ -304,3 +304,53 @@ def diverging_stacked_bar(data_frame: pd.DataFrame, **kwargs) -> go.Figure:
         fig.add_hline(y=0, line_width=2, line_color="grey")
 
     return fig
+
+
+@capture("graph")
+def lollipop(data_frame: pd.DataFrame, **kwargs):
+    """Creates a lollipop based on px.scatter.
+
+    A lollipop chart is a variation of a bar chart where each data point is represented by a line and a dot at the end
+    to mark the value.
+
+    Inspired by: https://community.plotly.com/t/how-to-make-dumbbell-plots-in-plotly-python/47762
+
+    Args:
+        data_frame: DataFrame for the chart. Can be long form or wide form.
+            See https://plotly.com/python/wide-form/.
+        **kwargs: Keyword arguments to pass into px.scatter (e.g. x, y, labels).
+            See https://plotly.com/python-api-reference/generated/plotly.scatter.html.
+
+    Returns:
+        go.Figure: Lollipop chart.
+    """
+    # Should we allow keyword arguments (kwargs) in this context, given the presence of multiple traces?
```

Comment:
@antonymilne - I am still a bit torn which code we should display in the visual-vocabulary here and what the requirements should be. I still prefer any code that can be easily adapted + follows data viz best practices. 

This sometimes interferes with the requirements we might have for any chart that should live in `vizro.charts` as already discussed. I have a couple of open questions here, which we can discuss here or offline.